docs/proposals/groups.md

```diff
@@ -0,0 +1,94 @@
+# Groups
+
+## Problem
+Project administrators need to be able to define groups so they can efficiently manage policy on their projects, however a project administrator is unlikely to be allowed to manage cluster-scoped group membership across all projects.  A single project administrator (or set of project administrators) is likely to own multiple related projects (think single department running multiple projects).  To allow cluster-scoped group definitions to be shared across multiple related projects, we'll introduce the concept of ClusterGroups (cluster-scoped sets of users), OrgGroups (scoped sets of users), and Organizations (things that reference Users and own OrgGroups).
+
+
+## Use Cases
+ 1.  I want to enable a subset of users to be able to organize sets of Users into OrgGroups, but I only want those changes to affect policy evaluation across some projects.
+ 1.  I want to be able to define some ClusterGroups that will affect policy evaluation across all projects.
+
+
+## Overview
+Organizations are cluster-scoped resources that contains a set of OrgGroups, a list of admins (users that can manage the groups), and a list of members (users that can be added to the OrgGroups).  ClusterGroups are cluster-scoped sets of users that are managed by a cluster-admin.  OrgGroups are organization scoped sets of users that are managed by the Organization.AdminUsers.  Projects may have at most one owning Organization.  OrgGroups may only be referenced in RoleBindings of Projects owned by the Organization.
```

Comment:
s/users that can be added to the OrgGroups/users that can be part of the OrgGroups

Members can be added and removed, that's why can be part is better wording imho.